src/cpp/session/modules/SessionRCompletions.R

```diff
@@ -1064,6 +1070,9 @@ assign(x = ".rs.acCompletionTypes",
          if (is.function(dollarNamesMethod))
          {
             allNames <- dollarNamesMethod(object)
+            
+            # check for custom helpHandler
+            helpHandler <- attr(allNames, "helpHandler")
```

Comment:
Let's not merge yet, I have one more think related to parameter help that
I'd like to nail down

On Mon, Sep 12, 2016 at 1:24 PM, Jonathan <notifications@github.com> wrote:

> In src/cpp/session/modules/SessionRCompletions.R
> <https://github.com/rstudio/rstudio/pull/780#discussion_r78415619>:
>
> > @@ -1064,6 +1070,9 @@ assign(x = ".rs.acCompletionTypes",
> >           if (is.function(dollarNamesMethod))
> >           {
> >              allNames <- dollarNamesMethod(object)
> > +
> > +            # check for custom helpHandler
> > +            helpHandler <- attr(allNames, "helpHandler")
>
> Nit: needs exact = TRUE
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/rstudio/pull/780/files/353071d3e4a64ab2c2547cbd7bbf85731b4383f7#r78415619>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXx3iKJGFBh8CruzDeEox8nqnedcxxks5qpYrlgaJpZM4J5Tzb>
> .
>